recipe/meta.yaml

```
@@ -52,9 +53,7 @@ test:
     {% set lib_idents = [ "Xext" ] %}
     {% for lib_ident in lib_idents %}
     - test -f $PREFIX/lib/lib{{ lib_ident }}.dylib  # [osx]
-    - test -f $PREFIX/lib/lib{{ lib_ident }}.so  # [linux]
-    - if not exist %PREFIX%/Library/lib/lib{{ lib_ident }}.dll.a exit /b 1  # [win]
-    - if not exist %PREFIX%/Library/bin/msys-{{ lib_ident }}-*.dll exit /b 1  # [win]
+    - test -f $PREFIX/lib/lib{{ lib_ident }}.so     # [linux]
```

```suggestion
    - test -f $PREFIX/lib/lib{{ lib_ident }}.so  # [linux]
    - if not exist %LIBRARY_LIB%/lib{{ lib_ident }}.a exit /b 1  # [win]
```
Same concerns about converting to `.lib` files and/or making `.dll`s.